flashinfer_trace/tests/references/test_top_k_top_p_sampling_from_probs_v262208.py

```
@@ -0,0 +1,142 @@
+        print("WARNING: CUDA not available, skipping test")
+        return False
+
+    torch.manual_seed(42)
+    probs, top_k, top_p = generate_random_inputs(batch_size, "peaked", device)
+
+    # Count frequencies for both implementations
+    ref_counter = torch.zeros(batch_size, VOCAB_SIZE, dtype=torch.int32, device=device)
+    fi_counter = torch.zeros(batch_size, VOCAB_SIZE, dtype=torch.int32, device=device)
+
+    print(f"Running {num_trials} trials to compare distributions...")
+    for trial in range(num_trials):
```

Ruff (0.15.4)

[warning] Loop control variable `trial` not used within loop body

Rename unused `trial` to `_trial`

(B007)